### ISSUE TITLE
Slapshot Mag - OUT NOW!

### ISSUE THREAD

**Issue by callaght** [Simbul/baker#1127]
Baker Master aae5f59a20
MagRocket 1.0

SLAPSHOT MAG!

Slapshot is a magazine about all things Ice Hockey and is published on Apple Newsstand AppStore.

https://itunes.apple.com/us/app/slapshot-mag/id653427456?ls=1&mt=8

Thank you for your help in my quest to publish my first MagRocket-Baker Newsstand app!  Thanks to: -

Simbul, nin9creative, folletto, freney,  nfeiglin, fturner19, MaverickWar,, DrByrnes, jmwatt, lifestylenetworker, Barney2324, thejoepeach, jeffreypharrington, Appmantx, mit1818, FunkeXMix

![slapshot-cover-issue1-3d-v2-720x714](https://f.cloud.github.com/assets/4885291/1137255/eb8aa710-1c58-11e3-96cf-a23431dfc83e.jpg)

![video snapshot](https://f.cloud.github.com/assets/4885291/1137256/00b1c376-1c59-11e3-8cc0-962f503f06e8.png)

http://www.youtube.com/watch?v=ISsXxWb8Eh4

**Comment by JamesMowatt** [comment]
Looks awesome! I will have a look tonight!

**Comment by qutehoot007** [comment]
Congrats.

**Comment by nin9creative** [comment]
Congrats :)

**Comment by fturner19** [comment]
Congrats you've done it, now lets get some downloads data ;)

**Comment by Appmantx** [comment]
Congrats! Just downloaded!! Looks great. How do you the page "Hockey After the lockout" That's really cool ,Two pages in one.

**Comment by callaght** [comment]
I hand the designer the copy text, the designer uses Abode InDesign to create the magazine and that's then handed back to me when its complete so I'm not sure how thats done I'm afraid!

I literally run it through In5 and upload it AWS S3.

**Comment by Appmantx** [comment]
Thanks Tom, I have used In5 too, but have had a problem exporting, I now 
get an error code in line 1319. Wait for an answer from Justin over @ 
Ajarproductions

**Comment by callaght** [comment]
Hope you get your error resolved soon @Appmantx

**Comment by nin9creative** [comment]
Have you had any issues with the layouts provided by in5?  I have a copy and have worked on some tests
and it doesn't seem to generate a 1 to 1 representation of the Indesign layout.  Even with goofing around with lots of 
the various export settings.

Andrew

**Comment by callaght** [comment]
I used: -

InDesign CS6.0 version 8.0
In5 version 1.4.20

With that combination I have been generally ok.  The only issue I had were the call-out circles in some of the articles being vertically lower down the page when exported than they were vertically positioned in InDesign.

Here are screenshots of the In5 settings that were used for Issue 1: -

![screen shot 2013-09-13 at 18 36 30](https://f.cloud.github.com/assets/4885291/1140185/1b2ecda2-1c9a-11e3-811a-fc8f963eede6.png)
![screen shot 2013-09-13 at 18 36 39](https://f.cloud.github.com/assets/4885291/1140188/272531fa-1c9a-11e3-9096-280079507f67.png)
![screen shot 2013-09-13 at 18 36 46](https://f.cloud.github.com/assets/4885291/1140193/2be2a664-1c9a-11e3-8a0d-b91913f35486.png)

**Comment by nin9creative** [comment]
Also, curious.  Did you use normal iPad size documents or retina sized?

Andrew

**Comment by callaght** [comment]
I think I had this discussion with the magazine designer - I think we went for Retina sized.

**Comment by nin9creative** [comment]
Ok, because that is another issue I had run into.  When exported with retina size indesign documents there
were weird issues with the page zooming.

Andrew

**Comment by callaght** [comment]
ah yes, this does ring a bell.  I think we had a problem when enabling zooming that the magazine only took up 1/4 of the screen.  For this reason zooming has been disabled to resolve this issue.

**Comment by nin9creative** [comment]
Yeah, that's the problem that I saw.  However I didn't try all the permutations of turning things on and off.

So turning off zooming resolves it?  Will need to try again! :)

Andrew

**Comment by callaght** [comment]
In my case I _think_ it did!

**Comment by callaght** [comment]
Is everyone able to download a copy of Slapshot Mag and read it?

I'm getting emails from people saying they can't download the mag :-(

**Comment by JamesMowatt** [comment]
#1133

**Comment by callaght** [comment]
Thank you for opening this @jmowatt

**Comment by Appmantx** [comment]
Tom, works fine for me.

**Comment by fturner19** [comment]
I encountered a problem when trying to read the magazine. The way it resolved itself is when I clicked restore purchases when I encountered the error. 
![img_20130913_144852](https://f.cloud.github.com/assets/280993/1143568/f4041446-1d46-11e3-98d1-9a62d46cb44a.jpg)

![1](https://f.cloud.github.com/assets/280993/1143559/d5019352-1d46-11e3-862f-2e3d89f0f8b3.png)
![2](https://f.cloud.github.com/assets/280993/1143567/ecfea972-1d46-11e3-8978-ccd8eb653c02.png)

Another thing you should address is the notification regarding future issues showing "Baker" should be changed to "Slapshot"!
Other than that everything works fine and I love the magazine content as well, keep up the great work. Success is within arms reach!

**Comment by callaght** [comment]
Thanks for checking @Appmantx 

@fturner19 - Thanks for feedback - this is very useful!  I'll post the sales data in a few days for ya!

**Comment by nin9creative** [comment]
Let me know if you notice anything odd with the MagRocket backend etc.  Interested if any errors are logged or the like.  Trying to get some feedback with production apps :)

Andrew

**Comment by callaght** [comment]
@nin9creative, I certainly will.  I've been watching the system log for a few days now and nothing has appeared - but if it does I'll post it here
